### ISSUE TITLE
installation fails for tensorflow-gpu

### ISSUE THREAD

**Issue by BrannonKing** [tensorflow/gnn#21]
When attempting to install from pip I get this error:
```
Could not find a version that satisfies the requirement tensorflow-cpu>=2.7.0 (from tensorflow-gnn) (from versions: 2.8.0rc0)
```
Admittedly, I have tensorflow-gpu installed. Is this library supposed to be specific to the cpu version?

**Comment by cloudlakecho** [comment]
Have you tried with Python 3.9 version?

When I read "setup.py" code, it looks like required [Python 3.9 or up](https://github.com/tensorflow/gnn/blob/main/setup.py#L195). This may be due to Tensorflow 2.7 supported only Python 2.9 and up.

**Comment by cjoverbay** [comment]
I am also having this problem on an m1 mac:
```

Requirement already satisfied: six in /opt/homebrew/Caskroom/miniforge/base/envs/tf_mac/lib/python3.9/site-packages (from tensorflow-gnn==0.1.0) (1.15.0)
ERROR: Could not find a version that satisfies the requirement tensorflow-cpu>=2.7.0 (from tensorflow-gnn) (from versions: none)
ERROR: No matching distribution found for tensorflow-cpu>=2.7.0
```

**Comment by mihirparadkar** [comment]
This should be fixed with the 0.2.0 upload to PyPI. Please open another issue if you encounter further problems with `pip install tensorflow_gnn`
